src/api/sqs/handlers/index.ts

```
@@ -3,3 +3,4 @@ export { emailMembershipPassHandler } from "./emailMembershipPassHandler.js";
 export { provisionNewMemberHandler } from "./provisionNewMember.js";
 export { sendSaleEmailHandler } from "./sendSaleEmailHandler.js";
 export { emailNotificationsHandler } from "./emailNotifications.js";
+export { createOrgGithubTeamHandler } from "./createOrgGithubTeam.js";
```

ESLint

[error] Unexpected use of file extension "js" for "./createOrgGithubTeam.js"

(import/extensions)